docs/east_ocr.md

```diff
@@ -101,12 +101,60 @@ The custom node east_ocr can be built inside a docker container via the followin
 This command will export the compiled library in `./lib` folder.
 Copy this `lib` folder to the same location with `CRNN_Tensorflow` and `east_icdar2015_resnet_v1_50_rbox`.
 
-
 ## OVMS Configuration File
 
+The configuration file for running the OCR demo is stored in [config.json](../src/custom_nodes/east_ocr/config.json)
+Copy this file along with the model files and the custom node library like presented below:
+```bash
+OCR
+├── config.json
+├── crnn_tf
+│   └── 1
+│       ├── frozen_graph.bin
+│       └── frozen_graph.xml
+├── east_fp32
+│   └── 1
+│       ├── model.bin
+│       └── model.xml
+└── lib
+    └── libcustom_node_east_ocr.so
+```
 
 ## Deploying OVMS
 
+Deploy OVMS with OCR demo pipeline using the following command:
+
+```bash
+docker run -p 9000:9000 -d -v ${PWD}/OCR:/OCR openvino/model_server --config_path /OCR/config.json --port 9000
+```
 
 ## Requesting the Service
 
+While the OVMS is started with the pipeline loaded, you can query it using an exemplary client [easr_orc_client.py](../example_client/east_ocr_client.py).
```

Comment:
"Examplary client (LINK) can be used to request pipeline deployed in previous step"